packages/edit-site/src/components/sidebar-navigation-screen-navigation-menus/constants.js

```diff
@@ -3,7 +3,7 @@
 // on apiFetch this query is limited to 100.
 export const PRELOADED_NAVIGATION_MENUS_QUERY = {
 	per_page: 100,
-	status: 'publish',
+	status: [ 'publish', 'draft' ],
```

Comment:
We need to amend the preloading in https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.3/navigation-block-preloading.php

Also can we add a link to that file as a code comment so folks are aware of this in future?

Reply:
I _think_ this change might mean we can actually only have a single preloaded request which would be excellent. Needs to be carefully tested though.